electron/src/services/uhk-device-provider.ts

```diff
@@ -0,0 +1,18 @@
+import { Provider } from '@angular/core';
+
+import { UhkDeviceService } from './uhk-device.service';
+import { UhkHidApiService } from './uhk-hid-api.service';
+
+export function uhkDeviceProvider(): Provider {
+    // HID API officially support MAC, WIN and linux x64 platform
+    // https://github.com/node-hid/node-hid#platform-support
+    if (process.platform === 'darwin' ||
```

Comment:
Let's rather use hidapi exclusively. It should support every platform we need. This way, we'll be able to test hidapi on every platform and ditch libusb eventually if everything works.

https://github.com/node-hid/node-hid#platform-support : 

> ia32, ARM, MIPSel and PPC64 platforms are known to work but are not currently part of our test or build matrix. ARM v4 and v5 was dropped from Node.js after Node v0.10.